src/main/resources/content/requirements/caLearnAboutCapabilityAssessment.json

```diff
@@ -0,0 +1,72 @@
+{
+  "breadCrumbs1": [
+    {
+      "text": "Dashboard",
+      "href": "/dashboard"
+    },
+    {
+      "text": "Choose a commercial agreement",
+      "href": "/projects/choose-agreement"
+    },
+    {
+      "text": "Procurement overview",
+      "href": "/projects/create-or-choose"
+    },
+    {
+      "text": "Learn about capability assessment",
+      "href": "#"
+    }
+  ],
+  "breadCrumbs2": [
+    {
+      "text": "Dashboard",
+      "href": "/dashboard"
+    },
+    {
+      "text": "Choose a commercial agreement",
+      "href": "/projects/choose-agreement"
+    },
+    {
+      "text": "Procurement overview",
+      "href": "/projects/create-or-choose"
+    },
+    {
+      "text": "Write and publish your requirements",
+      "href": "/ca/task-list"
+    },
+    {
+      "text": "Learn about capability assessment",
+      "href": "#"
+    }
+  ],
+
+  "title": "Learn about capability assessment",
+  "subtitle": null,
+  "events": [
+    {
+      "eventno": 1,
+      "eventTitle": "Project and collaborators",
+      "eventDescription": "You can name your project in this section. You can also change who leads the procurement or add more colleagues to your project to help you evaluate suppliers.",
+      "eventTask": [
+        {
+          "Task": 1,
+          "link": "/rfp/name-your-project",
```

Comment:
this pr includes ac and rfp?

Reply:
Only capability assessment related pages